substrate/frame/contracts/src/wasm/runtime.rs

```diff
@@ -791,10 +821,129 @@ impl<'a, E: Ext + 'a> Runtime<'a, E> {
 		let key = self.decode_key(memory, key_type, key_ptr)?;
 		let outcome = self.ext.get_storage_size(&key);
 
-		self.adjust_gas(charged, RuntimeCosts::ClearStorage(outcome.unwrap_or(0)));
```

Comment:
Seems to be a bug. Can it be fixed in this commit, or should it be addressed in a separate one

Reply:
Let's leave it in here.